SafeguardDevOpsServiceWix/Product.wxs

```diff
@@ -3,7 +3,8 @@
      xmlns:util="http://schemas.microsoft.com/wix/UtilExtension"
      xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
 
-  <Product Id="*" Name="SafeguardDevOpsService" Language="1033" Version="!(bind.fileVersion.DevOpsPluginCommon.dll)" Manufacturer="!(loc.CompanyName)" UpgradeCode="8a0194a6-6aed-441c-8130-a81b96490bd7">
+  <Product Id="*" Name="SafeguardDevOpsService" Language="1033" Version="!(bind.fileVersion.DevOpsPluginCommon.dll)" 
+           Manufacturer="!(loc.CompanyName)" UpgradeCode="8a0194a6-6aed-441c-8130-a81b96490bd7">
```

Comment:
You shouldn't need to make any changes to the wix installer.  Probably need to revert this.

Reply:
I have VS2019 to thank for that.  I was just trying to build the msi for initial deployment.  s/b reverted now